dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -121,16 +136,31 @@
       <artifactId>jackson-databind</artifactId>
       <version>2.9.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-yaml</artifactId>
+      <version>2.11.2</version>
+    </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
       <version>2.9.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jdk8</artifactId>
+      <version>2.11.2</version>
+    </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.datatype</groupId>
       <artifactId>jackson-datatype-jdk8</artifactId>
       <version>2.9.1</version>
```

Comment:
should you remove this line instead?

Reply:
There are components in CL that still depend on the `com.ibm.ws.com.fasterxml.jackson` project... which pulls in version `2.9.1` of the relevant Jackson libraries. We cannot remove them until we have updated the components in CL to point at the `2.11.2` version of these libraries... and this cannot be done until they are in OL master. So these changes need to be made in phases, requiring 3 PRs in total... of which this is just the first PR.